### ISSUE TITLE
Console error on every single page using Formie

### ISSUE THREAD

**Issue by clsse** [verbb/formie#355]
**Description**
I get a console error for every single page using Formie.

The error is:
`Uncaught TypeError: Cannot set property 'value' of null`

It refers to this script tag which seems to be auto-appended to the template:
```
<script>(function(){ document.getElementById("__JSCHK_6031f3cf45280").value = "6031f102634e1"; })();
window.FormieTranslations={"File {filename} must be smaller than {filesize} MB.":"File {filename} must be smaller than {filesize} MB.","Choose up to {files} files.":"Choose up to {files} files.","This field is required.":"This field is required.","Please select a value.":"Please select a value.","Please select at least one value.":"Please select at least one value.","Please fill out this field.":"Please fill out this field.","Please enter a valid email address.":"Please enter a valid email address.","Please enter a URL.":"Please enter a URL.","Please enter a number":"Please enter a number","Please match the following format: #rrggbb":"Please match the following format: #rrggbb","Please use the YYYY-MM-DD format":"Please use the YYYY-MM-DD format","Please use the 24-hour time format. Ex. 23:00":"Please use the 24-hour time format. Ex. 23:00","Please use the YYYY-MM format":"Please use the YYYY-MM format","Please match the requested format.":"Please match the requested format.","Please select a value that is no more than {max}.":"Please select a value that is no more than {max}.","Please select a value that is no less than {min}.":"Please select a value that is no less than {min}.","Please shorten this text to no more than {maxLength} characters. You are currently using {length} characters.":"Please shorten this text to no more than {maxLength} characters. You are currently using {length} characters.","Please lengthen this text to {minLength} characters or more. You are currently using {length} characters.":"Please lengthen this text to {minLength} characters or more. You are currently using {length} characters.","Unable to parse response `{e}`.":"Unable to parse response `{e}`."};
(function(){ document.getElementById("__JSCHK_6031f3cf4df3e").value = "6031f102634e1"; })();</script>
```

The part throwing the error is:
`document.getElementById("__JSCHK_6031f3cf45280").value = "6031f102634e1"; `

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.18 / latest
- Craft version: 3.6
- Multi-site: No

##### Update
I do want to point out that after testing, I noticed the error goes away if I disable the "Javascript" integration. Not sure if it's a problem with the integration and/or if you maintain that integration but did think it is worth noting.

**Comment by engram-design** [comment]
Seems to be an issue indeed with the Javascript integration, although it does seem odd that it can't find that input.

This integration adds a hidden input similar to `<input type="hidden" id="__JSCHK_603210a25c7d3" name="__JSCHK_7013547" value="6032109f3c1eb">`. Do you see a hidden input containing `JSCHK` at all on your page?

**Comment by clsse** [comment]
No, there is no such input, unfortunately!

**Comment by engram-design** [comment]
Very strange, as that JS and HTML is output together, and I can't seem to replicate it. See https://github.com/verbb/formie/blob/craft-3/src/integrations/captchas/Javascript.php#L74-L75

**Comment by clsse** [comment]
Hey @engram-design I disabled globally, reenabled the integration globally, and then enabled it on the one form and now the input **does** show up. But the ID doesn't match the ID that is being sought out by the script tag:

`<input type="hidden" id="__JSCHK_6032dbf34a6f9" name="__JSCHK_180945">`
`(function(){ document.getElementById("__JSCHK_6032dbf349073").value = "6032dbf3490ac"; })();`

The latter throws the error `Uncaught TypeError: Cannot set property 'value' of null`.

**Comment by engram-design** [comment]
Thanks for confirming that. Does your site use any form of caching?

**Comment by clsse** [comment]
Nothing special, no. Just craft caching out of the box. I did try clearing all caches before, and again just now, with no success.

**Comment by engram-design** [comment]
You mention Craft caching - do you mean template caching using the `{% cache %}` Twig tags? If so, I believe that might be the issue. I'll see what I can do.

**Comment by clsse** [comment]
No! No cache tags. I just mean the native out-of-the-box cache settings (in Utilities). Nothing that I manually put in place. Thank you for looking into this, let me know if you find anything and please don't hesitate to ask any further questions!

**Comment by engram-design** [comment]
Oh right - I don't think there should be any caching out of the box then, so that's odd. I've made a fix that might help with this in any case. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.18",
  "...": "..."
}
```

Then run `composer update`.

**Comment by clsse** [comment]
@engram-design Is this a relatively stable update? This would be live on a client's site which is launching next week so am a bit weary. Any ideas when it would be in a stable release? I may wait for that. (I'll probably test this locally for now but just wondering)

**Comment by engram-design** [comment]
It should be yes, but you're more than welcome to wait until a formal release, should be in the next 1-2 days.

**Comment by engram-design** [comment]
Fixed in [1.3.19](https://github.com/verbb/formie/releases/tag/1.3.19)
